packages/react/src/helpers/utils.ts

```diff
@@ -53,3 +53,10 @@ export const toErrorWithMessage = (maybeError: unknown): ErrorWithMessage => {
 export const getErrorMessage = (error: unknown) => {
   return toErrorWithMessage(error).message;
 };
+
+export const getFormDataFromEvent = (
```

Comment:
Created a separate one for React, because React has specific type for `React.FormEvent<HTMLFormElement>`.

Reply:
👍